nrf_modem/doc/sockets.rst

```diff
@@ -476,6 +476,38 @@ In this case, either security tag 4 or security tag 5 can be used for operations
 
    Using multiple security tags
 
+TLS traffic decryption
+**********************
+
+Starting from modem firmware v2.0.0, Transport Layer Security (TLS) traffic can be decrypted with Nordic tools if the TLS session is created using certificates stored to security tags ranging from :c:macro:`nrf_sec_tag_tls_decrypt_0` to :c:macro:`nrf_sec_tag_tls_decrypt_19`.
+
+.. important::
+   These security tags shall be used only for test and development purposes.
+
+Testing TLS traffic decryption 
+------------------------------
+
+Prerequisites:
+
+#. Make sure that the device is running an application that utilizes TLS. The TLS session shall be created using certificates stored to tags ranging from :c:macro:`nrf_sec_tag_tls_decrypt_0` to :c:macro:`nrf_sec_tag_tls_decrypt_19`.
+#. Make sure that the application has modem traces enabled. Information on modem traces and how to enable them is found in the :ref:`modem_trace` documentation.
+#. Make sure that the device is running modem firmware v2.0.0 and higher. Information on how to find the modem firmware version is found in the `Revision Identification +CGMR <https://docs.nordicsemi.com/bundle/ref_at_commands/page/REF/at_commands/general/cgmr_set.html>`_ documentation.
+#. Make sure that Wireshark is installed on your machine. Information on Wireshark and how to install it is found at the `Wireshark <https://www.wireshark.org/>`_ website.
```

Comment:
`#. The device programs an application that supports TLS. `

This sentence is not very clear to me because a device cannot program an application. Should it be `#. The device runs an application that supports TLS. ` instead?

Reply:
> The links should be added to link.txt.
> 
> For Revision Identification +CGMR - use the following two link: https://infocenter.nordicsemi.com/topic/ref_at_commands_nrf91x1/REF/at_commands/general/cgmr.html and https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/general/cgmr.html , depending on the DK you are using. We still refer to the infocenter docs not the Zoomin. Zoomin is in Beta phase.

I would use the one for the newest DK since the older DK will not support mfw v2.0.0 (and thus not support this feature).